### ISSUE TITLE
DL3008 bug when target-release is used

### ISSUE THREAD

**Issue by shaneog** [hadolint/hadolint#177]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

Lint passes.

### Actual behavior
Lint fails with message
```
Dockerfile:3 DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
```

### Steps to reproduce the behavior

Lint the below Dockerfile.
The `target-release` parameter `-t` or `--target-release` seems to be causing a fail for the DL3008 linter.
The package I am installing has its version specified.

---

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter 1.5.1-no-git
```
```bash
Haskell Dockerfile Linter v1.5.1-0-gd759349
```

**Dockerfile** (if relevant)

```Dockerfile
FROM debian:jessie

RUN set -e &&\
      echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list &&\
      apt-get update &&\
      apt-get install -y --no-install-recommends -t jessie-backports openjdk-8-jdk=8u131-b11-1~bpo8+1 &&\
      apt-get autoremove -qq -y --purge &&\
      apt-get clean &&\
      rm -rf /var/lib/apt/lists/*
```

**Additional environment details (OS, stack version, etc.)**

**Comment by lorenzo** [comment]
Thanks for reporting, we'll look into it shortly
